.github/workflows/fmt.yml

```diff
@@ -19,6 +19,10 @@ jobs:
           components: rustfmt
       - run: cargo fmt --all -- --check
   clippy_test:
+    strategy:
+      fail-fast: true
+      matrix:
+        features: ["web-axum", "web-rocket", "web-tide", "web-tower"]
```

Comment:
Are the other features like `sqlx-postgres` still running through clippy somehow?

Reply:
For future maintainability, I'm a bit worried because we are losing the clippy checks on the other features like `sqlx-postgres` and `loader`. Maybe we should add an extra clippy check which does all features except the frameworks?

Reply:
@chesedo we can create two steps to clippy check. One of web with this matrix and another one for the others.